src/Test/Perf/ConsumptionParser/Program.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Roslyn.Test.Performance.Utilities.ConsumptionParser
+{
+    internal class Program
+    {
+        public static void Main()
+        {
+            const string prefix = "PerfResults-";
+            const string share = @"\\mlangfs1\public\basoundr\PerfTraces\";
```

Comment:
Should these be command line args (or at least fetch these from the Jenkins master via http)?